Mathlib/NumberTheory/NumberField/Embeddings.lean

```diff
@@ -1005,6 +1007,74 @@ lemma card_eq_card_isUnramifiedIn [NumberField k] [IsGalois k K] :
       Finset.card (Finset.univ.filter <| IsUnramifiedIn K (k := k))ᶜ * (finrank k K / 2) := by
   rw [← card_isUnramified, ← card_isUnramified_compl, Finset.card_add_card_compl]
 
+noncomputable section InfiniteCompletion
+
+variable {K : Type*} [Field K] [NumberField K] (v : InfinitePlace K)
+
+/-- The normed field structure of a number field coming from the norm asociated to
+an infinite place. -/
+abbrev normedField : NormedField K :=
+  NormedField.induced _ _ v.embedding v.embedding.injective
+
+/-- The embedding associated to an infinite place is a uniform embedding. -/
+theorem uniformEmbedding : letI := v.normedField; UniformEmbedding v.embedding :=
+  letI := v.normedField; ⟨uniformInducing_iff_uniformSpace.2 rfl, v.embedding.injective⟩
+
+/-- The completion of a number field at an infinite place. -/
+def completion :=
+  letI := v.normedField; UniformSpace.Completion K
+
+namespace Completion
+
+instance : NormedField v.completion :=
+  letI := v.normedField; letI := UniformSpace.comap_completableTopField v.embedding;
+  UniformSpace.Completion.instNormedField K
+
+instance : CompleteSpace v.completion :=
+  letI := v.normedField; UniformSpace.Completion.completeSpace K
+
+instance : Inhabited v.completion := ⟨0⟩
```

Comment:
Is this needed?

Reply:
I don't use it specifically, but seemed like a standard instance to include in this case. Mainly, I saw that it was included for the finite completions [`IsDedekindDomain.HeightOneSpectrum.instInhabitedAdicCompletion`](https://leanprover-community.github.io/mathlib4_docs/Mathlib/RingTheory/DedekindDomain/AdicValuation.html#IsDedekindDomain.HeightOneSpectrum.instInhabitedAdicCompletion)